seq2science/scripts/qnorm.py

```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+import qnorm
+
+
+df = pd.read_csv(snakemake.input[0], comment="#", index_col=0, sep="\t")
+
+# cpm normalization
+df = df * 1_000_000 / df.sum(axis=0)
+
+# quantile normalize
+df_qn = qnorm.quantile_normalize(df)
+open(str(snakemake.output[0]), "w").write(
```

Comment:
nitpick: Since you named the output arguments, I would use those names here for clarity.
IIRC something like `snakemake.output$real`

Reply:
It's not a named output I believe, otherwise I would very much agree